test/letters/actions/letters.unit.spec.js

```diff
@@ -59,19 +59,30 @@ const getState = () => ({});
 
 describe('saveAddress', () => {
   const frontEndAddress = {
-    type: ADDRESS_TYPES.military,
+    addressOne: '123 Any Street',
+    addressThree: '',
+    addressTwo: 'Apt 102',
     city: 'APO',
+    countryName: 'USA',
     stateCode: 'AE',
-    countryName: 'USA'
+    type: ADDRESS_TYPES.military,
+    zipCode: '12345',
+    zipSuffix: ''
   };
 
   const successResponse = {
     data: {
       attributes: {
         address: {
-          type: frontEndAddress.type,
+          addressEffectiveDate: null,
```

Comment:
This gets returned as `null` when we save the address? I think I remember that's what you said, but I want to verify.

Reply:
Yeah...in theory it shouldn't matter anymore with these changes but that's what `vets-api` returned from the PUT.